front/src/components/pages/ChannelPage/ChannelPage.tsx

```diff
@@ -0,0 +1,149 @@
+              resolve([
+                {
+                  isPrivate: true,
+                  channelName: '42채팅방-1',
+                  joinCnt: 5,
+                  totalCnt: 5,
+                  adminUserIds: [0, 1, 5],
+                },
+                {
+                  isPrivate: false,
+                  channelName: '42채팅방-2',
+                  joinCnt: 5,
+                  totalCnt: 15,
+                  adminUserIds: [20, 21, 52],
+                },
+                {
+                  isPrivate: false,
+                  channelName: '42채팅방-3',
+                  joinCnt: 5,
+                  totalCnt: 10,
+                  adminUserIds: [10, 11, 15],
+                },
+                {
+                  isPrivate: true,
+                  channelName: '42채팅방-1',
+                  joinCnt: 5,
+                  totalCnt: 5,
+                  adminUserIds: [0, 1, 5],
+                },
+                {
+                  isPrivate: false,
+                  channelName: '42채팅방-2',
+                  joinCnt: 5,
+                  totalCnt: 15,
+                  adminUserIds: [20, 21, 52],
+                },
+                {
+                  isPrivate: false,
+                  channelName: '42채팅방-3',
+                  joinCnt: 5,
+                  totalCnt: 10,
+                  adminUserIds: [10, 11, 15],
+                },
+                {
+                  isPrivate: true,
+                  channelName: '42채팅방-1',
+                  joinCnt: 5,
+                  totalCnt: 5,
+                  adminUserIds: [0, 1, 5],
+                },
+                {
+                  isPrivate: false,
+                  channelName: '42채팅방-2',
+                  joinCnt: 5,
+                  totalCnt: 15,
+                  adminUserIds: [20, 21, 52],
+                },
+                {
+                  isPrivate: false,
+                  channelName: '42채팅방-3',
+                  joinCnt: 5,
+                  totalCnt: 10,
+                  adminUserIds: [10, 11, 15],
+                },
+                {
+                  isPrivate: false,
+                  channelName: '42채팅방-3',
+                  joinCnt: 10,
+                  totalCnt: 10,
+                  adminUserIds: [10, 11, 15],
+                },
+                {
+                  isPrivate: true,
+                  channelName: '42채팅방-31',
+                  joinCnt: 5,
+                  totalCnt: 10,
+                  adminUserIds: [10, 11, 15],
+                },
+              ]);
+            }, 1000);
+          });
+    return getChannelInfo();
+  };
+
+  return (
+    <Grid className={classes.root} container>
+      <Grid className={classes.menu} container>
+        <Grid item xs={9}>
+          <SubMenu menuItems={menuItems} onMenuChanged={onMenuChanged} />
+        </Grid>
+        <Grid className={classes.button} item xs={3}>
+          <Button href="/channel/add" width="auto">채널 추가</Button>
```

Comment:
```suggestion
          <Button href="/channel/add" width="small">채널 추가</Button>
```
auto로 하면 창 너비가 넓으면 너무 넓어보이는데 small로 픽스하는게 좋지 않을까요?